src/main/java/dk/xam/jbang/Main.java

```diff
@@ -216,6 +223,9 @@ private Integer doCall() throws IOException {
 			return 0; // quit(0);
 		} else if (completionRequested) {
 			return completion();
+		} else if (removeAlias) {
+			Settings.removeAlias(scriptOrFile);
+			return 0; // quit(0);
```

Comment:
you can do `jbang --clear-cache --init=test.java --edit-live myfile.java`  thus shouldn't `return 0`; here as no issue in doing `jbang --remove-alias='example' other.java`

Reply:
Ok

Reply:
(not that a really agree, but I guess that's the issue when using flags as a kind of command)

Reply:
yes, it has advantages and disadvantages for sure....i often ponder if should move to command layout instead but i can't my head around  that `jbang <script>` would be need to do `jbang run <script>` 

maybe for jbang2 one day ...

Reply:
Couldn't that be made a "shortcut" of some sort? That way you could perhaps do `jbang [run] <url | path>`. It would probably need a bit of custom handling where if no command is found it's assumed to be `run`.

Reply:
yeah, you could but it would be `[run] <url|path|alias|...>`  thus not exactly super clear.

Reply:
True, imagine someone having created an alias named `run` ;-)

Surely if we sleep on it some more we'll come up with a good idea

Reply:
One possible solution might be to not allow aliases in the shortcut version of the jbang command. So you'd get:

```
jbang <url|path>
jbang run <url|path|alias>
```

(and if you have a file called `run` you'll have to do `jbang ./run` to make the shortcut version work)

Reply:
Well, now that I think about it, you would still be able to use aliases with the shortcut version as well as long as there is no ambiguity. So running an alias called "run" is still possible using `jbang run run`.

You could say that the shortcut version is nice for humans typing in the command, while advising to use `jbang run` in scripts because there'll never be any ambiguity in that case. (Not even when installing a newer version of JBang with more commands)

Reply:
you are right! wanna create a separate issue to consider what a proper command structure would be before we declare jbang 1.0 ? :)

Reply:
#94